lib/Agrammon/Inputs.pm6

```diff
@@ -106,11 +106,12 @@ class X::Agrammon::Inputs::Distribution::AlreadyBranched is Exception {
 
 class X::Agrammon::Inputs::Distribution::BadSum is Exception {
     has Str $.what;
+    has $.sum;
     has Str $.taxonomy;
     has Str $.instance-id;
     has Str $.input-name;
     method message() {
-        "$!what.tclc() does not sum to 100 for instance $!instance-id of $!input-name in $!taxonomy"
+        "$!what.tclc() does sum to $!sum instead of 100 for instance $!instance-id of $!input-name in $!taxonomy"
```

Comment:
Very picky, but "sums to" is more idiomatic English than "does sum to"

Reply:
That's what I get of having an English man for reviews :-) ... appreciated and fixed.